### ISSUE TITLE
Multiple classes training

### ISSUE THREAD

**Issue by scpctshp** [plemeri/UACANet#9]
Hi, is it possible to use this model for multi-classes segmentation training? Thanks.

**Comment by plemeri** [comment]
Hello, sorry for the very late reply.
Yes, but you need some modifications. You need to change the output class number from 1 to the desired number, change the loss function for multi-class, change dataloader and custom transform functions.

**Comment by plemeri** [comment]
Also, you can find similar issue from [PraNet #9](https://github.com/DengPingFan/PraNet/issues/9).
